e2e/tests/ui-driven/src/mocks/geospatialMocks.ts

```diff
@@ -37,3 +37,165 @@ export const mockMapGeoJson: Feature = {
     "organisation-entity": "13",
   },
 };
+
+export const mockPropertyConstraints = {
+  sourceRequest:
```

Comment:
This `sourceRequest` is not accurately mocked - eg it includes `dataset=conservation-area` despite the later check that `!splitUrl?.includes("designated.conservationArea")` ? 

Did you simply fail to manually mock this or did you copy this from the frontend which then might point to a bug?